package.json

```diff
@@ -54,6 +54,7 @@
     "react-phone-number-input": "^2.3.21",
     "react-portal": "^4.1.1",
     "react-resizable": "^1.10.1",
+    "react-resize-detector": "^5.0.7",
```

Comment:
What are we using this for?

Reply:
It's being used inside <SectionalTabs /> to detect the browser resizing events.